lib/framework.php

```diff
@@ -29,6 +29,7 @@
 require APP_PATH.'lib/crypt.php';
 require APP_PATH.'lib/crypt_sodium.php';
 require APP_PATH.'lib/sodium_compat.php';
+require APP_PATH.'lib/scram.php';
```

Comment:
The diff mixes unrelated work. Please see https://en.wikipedia.org//wiki/Atomic_commit